libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -42,22 +43,18 @@ class MapboxDirectionsSession(
         routeOptions: RouteOptions,
         routesRequestCallback: RoutesRequestCallback?
     ) {
-        routes = emptyList()
         router.getRoute(routeOptions, object : Router.Callback {
             override fun onResponse(routes: List<DirectionsRoute>) {
                 this@MapboxDirectionsSession.routes = routes
                 routesRequestCallback?.onRoutesReady(routes)
-                // todo log in the future
```

Comment:
Bring the `TODO` comments back for when the `Logger` / `MapboxLogger` becomes available.

Reply:
👍